Source/AliveLib/Glukkon.cpp

```diff
@@ -2650,9 +2730,41 @@ void Glukkon::vOn_TLV_Collision_4404A0(Path_TLV* pTlv)
     }
 }
 
-void CC Glukkon::PlaySound_444AF0(unsigned __int8 /*sndIdx*/, __int16 /*volume*/, __int16 /*pitch*/, Glukkon* /*pGlukkon*/)
+SfxDefinition gameSpeak_554858[15] =
+{
+    { 0u, 8u, 66u, 127u, 0, 0 }, //HEY!

```

Comment:
you could add an enum for this and then change the call sites of PlaySound_GameSpeak_444AF0 to use it